core/arch/decode.h

```diff
@@ -323,6 +323,8 @@ enum {
                            * EVEX.LL'.
```

Comment:
> Please note that vpbroadcastb, vpbroadcastw, vpbroadcastd, vpbroadcastq supports an
additional general register version in AVX-512.

grammar: s/supports/support/ (or s/that/that each of /)

Reply:
Fixed.